src/commands/dev/lib/dev-router.ts

```diff
@@ -10,6 +10,25 @@ export default function(reqPath = '', routes?: RouteConfig[]): RouteResult {
   let found: RouteResult | undefined;
   const { pathname: reqPathname = '/' } = url.parse(reqPath);
 
+  const resolveRouteParameters = (str: string, dict: {keys: any, match: any}) => {
```

Comment:
Let's avoid `any` here. I think `keys: string[]` but not sure about `match`. Can you identify the type?

Reply:
@styfle i will have to check.

`const match = matcher.exec(reqPathname);`

Reply:
I added a suggestion below